crates/storage/src/schema/revision_0045.rs

```diff
@@ -0,0 +1,13 @@
+use anyhow::Context;
+
+pub(crate) fn migrate(tx: &rusqlite::Transaction<'_>) -> anyhow::Result<()> {
+    tx.execute_batch(
+        r"
+ALTER TABLE block_headers RENAME COLUMN gas_price TO eth_l1_gas_price;
+ALTER TABLE block_headers ADD COLUMN strk_l1_gas_price BLOB DEFAULT NULL;
```

Comment:
Are both L1 gas prices?

Reply:
I guess yes. And the name of the filed is `strk_l1_gas_price` in the JSON from the feeder gateway.